src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/ResultsViewExpansion.cs

```diff
@@ -176,5 +232,56 @@ private EvalResultDataItem CreateResultsViewRow(DkmInspectionContext inspectionC
                 editableValue: null,
                 inspectionContext: inspectionContext);
         }
+
+        private EvalResultDataItem CreateResultsViewRow(DkmInspectionContext inspectionContext, string name, Type declaredType, DkmClrValue value, Formatter formatter)
```

Comment:
Could we just call this overload of CreateResultsViewRow from CreateResultsOnlyRow (and then remove anything that's no longer needed from the other overload of CreateResultsViewRow)?  It's always seemed odd to me that if you do "e, results", we say the Value is "Expanding the Results View will enumerate the IEnumerable".  It seems like this new behavior would be better in that case as well.